src/cephadm/box/docker-compose.yml

```diff
@@ -7,39 +7,52 @@ services:
       - CEPH_BRANCH=master
     image: cephadm-box
     # probably not needed with rootless Docker and cgroups v2
-    privileged: true
-    # cap_add:
-    #   - SYS_ADMIN
-    #   - NET_ADMIN
-    #   - SYS_TIME
-    #   - MKNOD
+    # privileged: true
+    cap_add:
+      - SYS_ADMIN
+      - NET_ADMIN
+      - SYS_TIME
+      - SYS_RAWIO
+      - MKNOD
+      - NET_RAW
+      - SETUID
+      - SETGID
+      - CHOWN
+      - SYS_PTRACE
+      - SYS_TTY_CONFIG
```

Comment:
Can we also add `CAP_AUDIT_WRITE` and `CAP_AUDIT_CONTROL` to this? 

When I was testing it on my machine, it would fail at the bootstrapping seed step, specifically at the part where it tries to add the bootstrap host. I did `box -v cluster sh` and went into the seed, and noticed that my sshd logs had `fatal: linux_audit_write_entry failed: Operation not permitted`. After, I added `CAP_AUDIT_WRITE` and then I saw this issue in podman that matched mine: https://github.com/containers/podman/issues/13012

The solution was to add both `CAP_AUDIT_WRITE` and `CAP_AUDIT_CONTROL`. After I added those to the docker-compose, my bootstrap worked